mercury/tests/test_measurement.py

```diff
@@ -75,16 +75,25 @@ def post_defect_data(self):
     def test_Radio_Receiver_POST_Event_Not_Exist(self):
         response = self.client.post(reverse(self.post_url, args=[self.uuid2]))
         self.assertEqual(404, response.status_code)
+        self.assertTrue("Event uuid not found" in str(response.content))
+
+    @mock.patch("ag_data.models.AGEvent.objects.get", fake_event)
+    def test_Radio_Receiver_POST_Sensor_Not_Exist(self):
```

Comment:
Could you also add a test for when the sensor exists, and verify that the data stored in the measurement table is indeed using the correct sensor, event, etc.?

Reply:
I think this case is included in the test_Radio_Receiver_POST_Event_Success. Since we test all errors in other tests if sensors and events don't exist, they exist when the test_Radio_Receiver_POST_Event_Success passes.

Reply:
I believe `test_Radio_Receiver_POST_Event_Success` only tests the response status code. So we don't know if the API actually does what we expect it to do, does that make sense?